lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -1156,6 +1168,7 @@ func TestBuilder_Build_MMEPool(t *testing.T) {
 			CloudSubscriberdbEnabled: false,
 			AttachedEnodebTacs:       nil,
 			NatEnabled:               true,
+			SentryConfig:             &lte_mconfig.SentryConfig{},
```

Comment:
Here and throughout: just have this be implicitly nil

Reply:
Took these placeholders out.